internal/yunikorn/sync.go

```diff
@@ -0,0 +1,230 @@
+package yunikorn
+
+import (
+	"context"
+	"fmt"
+	"github.com/G-Research/yunikorn-history-server/internal/util"
+	"sync"
+
+	"github.com/apache/yunikorn-core/pkg/webservice/dao"
+	"github.com/google/uuid"
+)
+
+// startup performs all necessary steps to load up the current state of the cluster
```

Comment:
Looks like an old-comment  that can be deleted?

Reply:
indeed